### ISSUE TITLE
FSE: Navigation block has the toolbar on top of the block

### ISSUE THREAD

**Issue by Robertght** [Automattic/wp-calypso#64396]
### Quick summary

Hey!

In FSE, the toolbar covers the block itself, so we cannot select a specific menu or make edits on it.

This happens only in FSE, not in the Pages/Posts editor.

### Steps to reproduce

1. Activate an FSE theme
2. Go to Appearance > Editor
3. Add or try to modify an existing navigation block

### What you expected to happen

To be able to manage the menu easily

### What actually happened

![image](https://user-images.githubusercontent.com/6737345/172318626-58152ac2-597b-46f9-a164-63fee89354d2.png)

### Context

Discovered this while testing another PR

Slack ref: p1654278757904299-slack-C01DF571R5Y

cc @Automattic/cylon for further visibility

I couldn't replicate this on self-hosted

### Simple, Atomic or both?

Simple, Atomic

### Theme-specific issue?

FSE themes

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

No but the platform is still usable

**Comment by Robertght** [comment]
Core issue: https://github.com/WordPress/gutenberg/issues/41575

**Comment by Addison-Stavlo** [comment]
While the original screen capture above shows the toolbar's bottom position conflict with submenu items (as reported in core here https://github.com/WordPress/gutenberg/issues/40382#issuecomment-1102272035) 

This issue describes a new regression in block toolbar behavior, causing obstruction of any selected block near the top of the editor canvas. I opened a core issue here as well for this regression - https://github.com/WordPress/gutenberg/issues/41575

**Comment by ccwalburn** [comment]
Another case reported here: 35430108-hc

**Comment by hacchism** [comment]
Another case: 36124777-hc (the user didn't mention it actually but I noticed it while checking the user's Site Editor)

**Comment by khristiansnyder** [comment]
I've noticed this issue for a while and it is making it very difficult to use the block or add a new menu to FSE sites.

**Comment by tanjoymor** [comment]
Just to note that the work around for this is to pin the toolbar to the top. Click the 3-dots contextual menu in the top right corner and choose Top Toolbar.

**Comment by youbrokesomething** [comment]
There have been a couple of improvements to this since Gutenburg v14.0, the latest is in Gutenburg v14.3 and we should retest to see if this is fixed when that's out.

**Comment by happychait** [comment]
Stumbled upon this issue while trying to update [dotcom sub-menu support guide.](https://wordpress.com/support/menus/create-drop-down-menus/#create-a-drop-down-menu-in-the-site-editor)

Just want to note for myself, and for posterity, the core issues tracking sub-menu usability issues are the following:

https://github.com/WordPress/gutenberg/issues/40382 linked  from this comment: https://github.com/WordPress/gutenberg/issues/36977#issuecomment-1102277628 

The Core issue mentioned in the [above comment](https://github.com/Automattic/wp-calypso/issues/64396#issuecomment-1148634980) is for the top-level parent menu items, which has since been fixed 🎉

**Comment by Greatdane** [comment]
https://github.com/WordPress/gutenberg/issues/41575 has been closed, but it may be getting fixed via https://github.com/WordPress/gutenberg/issues/40382

**Comment by cuemarie** [comment]
📌 ACTIONS
- Due to the age of this issue, I've downgraded the priority from `High` to `Normal` to better reflect its status, as this is primarily waiting on Core fixes, and hasn't caused a ton of support volume while we wait for that update.

**Comment by annezazu** [comment]
Closing this out as the original problem was fixed in GB https://github.com/WordPress/gutenberg/issues/40382 with this PR: https://github.com/WordPress/gutenberg/pull/62711
